hbase-common/src/main/java/org/apache/hadoop/hbase/util/PrettyPrinter.java

```diff
@@ -184,7 +184,12 @@ private static long humanReadableIntervalToSec(final String humanReadableInterva
       hours = matcher.group(6);
       minutes = matcher.group(8);
       seconds = matcher.group(10);
+    } else {
+      LOG.warn(
+        "Given interval value : {} is not a number and not match human readable format, TTL will set to 0.",
```

Comment:
"Given interval value '{}' is not a number and does not match human readable format, TTL will be set to 0"

Reply:
Oh, we should not explicitly say 'TTL' here? This is a general method for parsing interval...

Reply:
Thanks for reviwing...
this method just used to convert TTL, it is a private method and just return ttl...although it's name means not just for TTL...
maybe we should change this LOG to
Given interval value : {} is not a number and not match human readable format, value will set to 0.
is it right? i will change it...

Reply:
Let's correct the grammar...

"Given interval value '{}' is not a number and does not match human readable format, value will be set to 0"

Reply:
You still missed two words...

"Given interval value '{}' is not a number and **_does_** not match human readable format, value will **_be_** set to 0"

Reply:
> You still missed two words...
> 
> "Given interval value '{}' is not a number and **_does_** not match human readable format, value will **_be_** set to 0"

sorry, i forget, i pushed again fixed it.